web/src/components/MalteHelpForm.tsx

```diff
@@ -0,0 +1,196 @@
+import React, { ReactElement, SyntheticEvent, useCallback, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+
+import { SecurityIcon, SupportIcon } from '../assets'
+import Icon from '../components/base/Icon'
+import Button from './base/Button'
+import InputSection from './base/InputSection'
+import RadioElement, { RadioGroup } from './base/RadioElement'
+
+const Note = styled.div`
+  display: flex;
+  padding-bottom: 10px;
+`
+
+const StyledIcon = styled(Icon)`
+  width: 25px;
+  height: 25px;
+  padding-right: 20px;
+  flex-shrink: 0;
+`
+
+const Divider = styled.hr`
+  margin: 12px 0;
+  background-color: ${props => props.theme.colors.textSecondaryColor};
+  height: 1px;
+  border: none;
+`
+
+const SubmitButton = styled(Button)`
+  background-color: ${props => props.theme.colors.textSecondaryColor};
+  color: ${props => props.theme.colors.backgroundColor};
+  border-radius: 100px;
+  width: fit-content;
+  align-self: center;
+  padding: 15px;
+  box-shadow:
+    0 2px 4px rgb(0 0 0 / 15%),
+    0 2px 4px rgb(0 0 0 / 15%);
+
+  :hover {
+    background-color: ${props => props.theme.colors.textSecondaryColor}D9;
+  }
+
+  :active {
+    box-shadow: none;
+  }
+`
```

Comment:
Why did you use custom styling here instead of directly applying it int the `TextButton` component and using that?

Reply:
This is because the form already implements #2497, which is not yet merged yet. See design in figma -> https://www.figma.com/file/EiRpMhSqscuaChGdhNVZQ4/MALTE-App?type=design&node-id=3-22&mode=design&t=CoIwpX4BiwJXzTlC-0
However if you really dislike using a coustom styling I can also add a comment in #2497

Reply:
I think we should either change the styling for all buttons or for none. Otherwise we have different styles in different places of which at least one is not reusable/other people won't know about :see_no_evil: